helm/polaris/README.md

```diff
@@ -48,7 +48,7 @@ A Helm chart for Apache Polaris (incubating).
 
 ### Optional
 
-When using EclipseLink backed metastore a custom `persistence.xml` is required, a Kubernetes Secret must be created for it. Below is a sample command:
+When using a custom `persistence.xml`, a Kubernetes Secret must be created for it. Below is a sample command:
```

Comment:
This looks like a change unrelated to the purpose of this PR :thinking:

Reply:
Good find! This must have come from a bad `helm-docs` run, let me fix this.